AFM: Add prereqs for using the BAT CLI on Windows 10 (#33)
mulesoft/docs-functional-monitoring: modules/ROOT/pages/bat-install-task.adoc

The Blackbox Automated Testing (BAT) CLI wrapper is a command line wrapper that you install to execute API tests.

== Prerequisite for Linux and MacOS Systems

> Before installing the BAT CLI, ensure that AdoptOpenJDK's OpenJDK 8 is installed on your system.

Reviewer: Is the company name important? It's just that the two together are a bit hard on the eye. Well, my eye, anyway.
```suggestion
Before installing the BAT CLI, ensure that OpenJDK 8 is installed on your system.
```

== Prerequisites for Windows 10 Systems

> . Install Windows Subsystem Linux (WSL)

Reviewer: I looked it up:
```suggestion
. Install Windows Subsystem for Linux (WSL)
```

+

> The instructions are at https://docs.microsoft.com/en-us/windows/wsl/install-win10.

Reviewer: For the sake of information maintenance, I wonder if we should not say only...:
```suggestion
Installation instructions are on the Microsoft website.
```
OR
```suggestion
Installation instructions are available at https://docs.microsoft.com.
```

> . Install Java 8, cURL, and unzip in the Linux distribution that you installed in your WSL.

Reviewer: ```suggestion
. Install Java 8, install cURL, and unzip the Linux distribution that you installed in your WSL.
```
OR do you mean this:
```suggestion
. Unzip the Linux distribution that you installed in your WSL and install into it Java 8 and cURL.
```

Reviewer 2: Hi, the required tools are:
- Java 8
- curl
- unzip (this is the name of the cli tool that unzips .zip files in linux)

The 3 of them must be installed in the WSL distro.

> . Install Java 8, cURL, and unzip installed in the Linux distribution that you installed in your WSL.

Reviewer 2: Should we change `... and unzip installed in the Linux... ` for `... and unzip in the Linux ...`?

> . Add the `bat` executable to your `PATH` environment variable.

Reviewer: ```suggestion
. Add the `bat` executable to your `PATH` environment variable:
```

> .. Open the `~/.bashrc` file (if you are using `bash`) or the `~/.zshrc` file (if you are using `zsh`) for editing. For example, to use the `vi` editor, enter this command:

Reviewer: ```suggestion
.. Open the `~/.bashrc` file (if you are using `bash`) or the `~/.zshrc` file (if you are using `zsh`) for editing. +
For example, to use the `vi` editor, enter the appropriate version of these two commands:
* `vi ~/.bashrc`
* `vi ~/.zshrc`
```
I suggest this because we do not want to join two syntax examples with one narrative word, such as "or" here, or a fragment.

+
`vi ~/.bashrc`
+
or
+

> `vi ~/.zshrc`

Reviewer: See above.

. Install Windows Subsystem Linux (WSL)
+
The instructions are at https://docs.microsoft.com/en-us/windows/wsl/install-win10.

> . Install Java 8 and cURL in the Linux distribution that you installed in your WSL.

Reviewer 2: `unzip` it's also a tool required to be installed. The correct requirement should be: `Install Java 8, cURL and unzip in the Linux distribution that you installed in your WSL.`
